opengcs/.dockerignore

```diff
@@ -0,0 +1,10 @@
+Dockerfile
```

Comment:
I don't think we need this or .gitignore, .travis.yml , CODEOWNERS, or the Dockerfile (which was already super out of date/depracated before 😅)

Reply:
This PR was mostly just getting the code and history in here and we could actually place things where they belong/delete things shortly after. If we'd rather do everything all in here that's fine with me too, it'll just be harder to review

Reply:
I think it would be a mistake to pull everything in as-is. We should first have a commit that reorganizes opengcs to align with hcsshim's organization.

Reply:
Oh, I see this is dumping everything into `/opengcs` rather than the root. That makes it a little easier to work with, but I still would prefer we not merge the repos until we understand how the code organization will be reconciled.

Reply:
Was hoping to figure that out either here or internally. Which would we prefer?

Reply:
Which what would we prefer?

Reply:
Meant do we want to discuss how to layout the code here or on a call/teams. I'll start a thread